src/main/java/formflow/library/FileController.java

```diff
@@ -218,55 +205,59 @@ public ResponseEntity<?> upload(
    * @param fileId               The id of an uploaded file that should be deleted
    * @param returnPath           The path to the page that they came from
    * @param dropZoneInstanceName The drop zone instance used to get the user file name
+   * @param flow                 The name of the current (active) flow
    * @param httpSession          The current HTTP session
    * @return ON SUCCESS: Returns a RedirectView to the returnPath
    * <p>ON FAILURE: Returns a RedirectView to the 'error' page</p>
    */
   @PostMapping("/file-delete")
-  RedirectView delete(
+  public RedirectView delete(
```

Comment:
why the public?

Reply:
It's a public method so just being explicit about it, thought it's clearly not necessary. Other methods in this file are public as well (line 279).

Reply:
I've been wondering about the need to include method scope (public/protected/private).  Do you know and have documentation on the best practice for this using java 17? I couldn't find anything, so I just kept being explicit so I don't fall into a trap where the default scope is different then I think it is.

Reply:
Perhaps you already knew all of this...
In reading [this page](https://www.geeksforgeeks.org/public-vs-package-access-modifiers-in-java) that if you don't include the scope, the method is defaulted to public w/in its package.  It gives package-level access to this function. 

If we are explicit and put "public" there, then the method is accessible from anywhere, not just w/i the package. 

Given this, it doesn't make much of a difference if public is here or not - I think all things in our codebase should be able to access this method (not that they really need to, though).  

I was adding them to make the file more consistent, because some methods had it and some didn't.

Reply:
That's my understanding as well, that the absence of an access modifier indicates package-private, which as you stated gives access only to other classes within the same package.

I don't have much in the way of documentation but do see that Springs own example includes the public:
https://docs.spring.io/spring-framework/reference/web/webmvc/mvc-controller.html

Maybe it makes sense to make this change across the board for consistency sake?

Reply:
I agree - something we should do for consistencies sake.  I made a ticket to track this work: https://www.pivotaltracker.com/story/show/186243988